src/java.desktop/share/classes/javax/swing/JTree.java

```diff
@@ -3310,45 +3310,45 @@ private int[] getModelIndexsForPath(TreePath path) {
         if(path != null) {
             TreeModel   model = getModel();
             int         count = path.getPathCount();
-            int[]       indexs = new int[count - 1];
+            int[]       indices = new int[count - 1];
             Object      parent = model.getRoot();
 
             for(int counter = 1; counter < count; counter++) {
-                indexs[counter - 1] = model.getIndexOfChild
+                indices[counter - 1] = model.getIndexOfChild
                                    (parent, path.getPathComponent(counter));
                 parent = path.getPathComponent(counter);
-                if(indexs[counter - 1] < 0)
+                if(indices[counter - 1] < 0)
```

Comment:
let's add space after `if`

Reply:
Better not: if you look at the code around it, there's no space between a keyword and the opening parenthesis. So the code looks consistent. Changing this `if` will make the code inconsistent.

Reply:
@turbanoff I'll go with @aivanov-jdk's suggestion here. Also, this is a PR created by an automation tool. I'd really prefer not to do any manual intervention, if possible, and at the very least not spelling related.